### PR TITLE
Novica veljanoski login

### PR DESCRIPTION
**Pull request by 173038**

### CODE REVIEW

File: project-novica/demo/src/main/java/com/example/demo/controller/AuthenticationController.java [hunk 1]
```diff
@@ -41,28 +42,65 @@ public String showRegistrationForm(Model model)
...
+        if (res.size() == 0)
+        {
+            return ResponseEntity.ok("success");
+        } else
+        {
+            return ResponseEntity.badRequest().body(String.join(", ", res));
+        }
+    }
+
+//    @PostMapping("/login")
```
Comment (aleksandar-bujaroski-armedia) [inline project-novica/demo/src/main/java/com/example/demo/controller/AuthenticationController.java]: Remove commented code

File: project-novica/demo/src/main/java/com/example/demo/controller/AuthenticationController.java [hunk 2]
```diff
@@ -41,28 +42,65 @@ public String showRegistrationForm(Model model)
...
 
+    @RequestMapping(path = "/login", method = RequestMethod.POST)
+    @ResponseBody
+    public ResponseEntity<String> logIn(@RequestBody UserDTO userDTO)
+    {
+        List<String> res = authenticationService.login(userDTO.getEmail(), userDTO.getPassword());
+        if (res.size() == 0)
+        {
+            return ResponseEntity.ok("success");
+        } else
```
Comment (aleksandar-bujaroski-armedia) [inline project-novica/demo/src/main/java/com/example/demo/controller/AuthenticationController.java]: Move else to next line to be aligned with if

File: project-novica/demo/src/main/java/com/example/demo/dto/UserDTO.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package com.example.demo.dto;
+
+
+public class UserDTO {
```
Comment (aleksandar-bujaroski-armedia) [inline project-novica/demo/src/main/java/com/example/demo/dto/UserDTO.java]: Fix code formatting

File: project-novica/demo/src/main/resources/static/javascript.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+function logIn() {
+    var UserDTO = {};
```
Comment (aleksandar-bujaroski-armedia) [inline project-novica/demo/src/main/resources/static/javascript.js]: Rename variable name to userDTO and rename the file to some proper name.
Example: loginUser.js

**Review by aleksandar-bujaroski-armedia — CHANGES_REQUESTED** [review]

File: project-novica/demo/src/main/resources/static/javascript.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+function logIn() {
```
Comment (aleksandar-bujaroski-armedia) [inline project-novica/demo/src/main/resources/static/javascript.js:1]: rename this file to more descriptive name:
Example:
login.js

**Review by aleksandar-bujaroski-armedia — CHANGES_REQUESTED** [review]

**Review by dejannastevski-armedia — APPROVED** [review]

**Pull request merged**